server/tests/db_token_test.py

```diff
@@ -0,0 +1,21 @@
+import pytest
+from sqlalchemy import select
+
+from .fixtures.deploy_db import async_minimum
+from npg.porchdb.models import Token
+
+@pytest.mark.asyncio
+async def test_token_creation(async_minimum):
+    'Test token creation from a fixture'
+
+    result = await async_minimum.execute(
+        select(Token)
+    )
+    tokens = result.scalars().all()
+    assert len(tokens) == 2
+    for t in tokens:
+        assert t.token != None
+        assert len(t.token) == 32
+        assert t.date_issued != None
```

Comment:
`t.date_issued is not None`. Using `is` is less likely to go wrong than `==` or `!=`